packages/components/atoms/f-form-field/src/components/FormDropdown.vue

```diff
@@ -70,6 +70,9 @@ export default {
 </script>
 
 <style lang="scss" module>
+$form-input-bg                            : $color-container-default;
```

Comment:
Should we be using this as the background colour for form elements when not `:hover`ed? I can't see this used anywhere apart from in the definition for `$form-input-bg--hover`, in which case, defining this separately is probably not necessary?

Reply:
Good point 👍

Reply:
@benwhite-deltas I've now removed this, thanks! 😄